lib/upload_system_log.pm

```diff
@@ -73,22 +73,19 @@ sub upload_supportconfig_log {
     }
     $args{options} //= '';
     $args{timeout} //= 600;
-    script_run("supportconfig -t . -B $args{file_name}", $args{timeout});
-    #FOR S390X LPAR
-    if (is_s390x) {
```

Comment:
Why remove the condition for s390?

Reply:
this is the reason on s390x we can't get the log file, for current supportconfig it will create all files that begin with scc_ no scc_supportconfig dir anymore. if we didn't issue dir name it will generate dir like/var/log/scc_seattle13_220311_0204